### PR TITLE
Edit-delete

### PR DESCRIPTION
**Pull request by prajapati-priyanka**
Things were added:
 - Notes edit functionality
 - Notes deleted from notes page goes to trash page.
 - Notes can restored to notes page from trash page.
 - Notes can be deleted from trash page.
[Preview](https://deploy-preview-4--jovial-lokum-1545ac.netlify.app/)
 
> **NOTE:**  As no notes are showing in the starting so will add one note to show as notes page gets loaded in my next PR.  Also, I will work on responsiveness in the end.

### CODE REVIEW

**Review by ravikumar1002 — APPROVED** [review]
UI looks awesome.
Code is clean and easy to read
looks good to merge

File: src/components/modal/createNoteModal/CreateNoteModal.jsx [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+                  className="select"
+                  name="priority"
+                  value={noteContent.priority}
+                  onChange={(e) =>
+                    setNoteContent((prevContent) => ({
+                      ...prevContent,
+                      priority: e.target.value,
+                    }))
+                  }
+                >
```
Comment (mariyasada) [inline src/components/modal/createNoteModal/CreateNoteModal.jsx:107]: you can directly create on function for change handler and save name and value in object like function chnageHandler(e){ const {name,value}=e.target; setNoteContent ((prevContent) => ({
                      ...prevContent,[name]:value})) like this

**Review by mariyasada — APPROVED** [review]
This code looks good to me just one another suggestion after clicking the save note button the pop-up box gets closed

File: src/components/modal/createNoteModal/CreateNoteModal.jsx (hunk 1 above)
Comment (prajapati-priyanka) [reply src/components/modal/createNoteModal/CreateNoteModal.jsx:107]: okay.

**Pull request merged**